index.js

```diff
@@ -68,4 +70,96 @@ window.addEventListener('scroll', recolorScrolledMenu);
 // TODO refactor this to update less often than every resize
 window.addEventListener('resize', recolorScrolledMenu);
 
-// HERO FUNCTIONALITY
+// ---
+// BEGIN FAQ FUNCTIONALITY
+// ---
+
+const questionArray = Array.from(document.querySelectorAll('.faq__question-container'));
+const showMoreButtonArray = Array.from(document.querySelectorAll('.faq__show-button'));
+
+// Individual question listeners
+function setQuestionListeners() {
+  questionArray.forEach(item => {
+    const toggle = item.querySelector('.faq__question-toggle');
+    const question = item.querySelector('.faq__question-subcontainer');
+    const answer = item.querySelector('.faq__answer');
+
+    question.addEventListener('click', () => {
+      const questionOpen = question.classList.contains('faq__question-subcontainer_open');
+      if (!questionOpen) {
+        question.classList.add('faq__question-subcontainer_open');
+        answer.classList.add('faq__answer_open');
+        toggle.src = './images/faq-close.png';
+      } else {
+        question.classList.remove('faq__question-subcontainer_open');
+        answer.classList.remove('faq__answer_open');
+        toggle.src = './images/faq-open.png';
+      }
+    });
+  });
+}
+
+// Show more button listeners
+function setShowMoreListener() {
+  showMoreButtonArray.forEach(button => {
+    const secondaryQuestionArray = Array.from(button.parentElement.querySelectorAll('.faq__question-container_group_secondary'));
+    const tertiaryQuestionArray = Array.from(button.parentElement.querySelectorAll('.faq__question-container_group_tertiary'));
+    const extendedSecondaryQuestionArray = [...secondaryQuestionArray, ...tertiaryQuestionArray];
+    let rev = 0;
```

Comment:
Just an opinion, I don't quite understand what 'rev' stands for(at first glance). Using the full word here could be better. Like 'reversedItems', 'hiddenItems', etc. but I already know what it means so it's fine.